internal/zinc-ivy-integration/src/main/scala/sbt/internal/inc/ZincComponentCompiler.scala

```diff
@@ -40,14 +40,12 @@ private[sbt] object ZincComponentCompiler {
   private[sbt] def getDefaultBridgeModule(scalaVersion: String): ModuleID = {
     def compilerBridgeId(scalaVersion: String) = {
       scalaVersion match {
-        case sc if (sc startsWith "2.10.")       => "compiler-bridge_2.10"
-        case sc if (sc startsWith "2.11.")       => "compiler-bridge_2.11"
-        case sc if (sc startsWith "2.12.")       => "compiler-bridge_2.12"
-        case "2.13.0-M1"                         => "compiler-bridge_2.12"
-        case sc if (sc startsWith "2.13.0-pre-") => "compiler-bridge_2.13.0-M5"
-        case sc if (sc startsWith "2.13.0-M")    => "compiler-bridge_2.13.0-M5"
```

Comment:
I'd keep this one: 1 extra line of code to allow future zinc/sbt's to build the milestone releases of scala 2.13.

Reply:
2.13.0-M2~5 should compile fine with 2.13.0-RC1 bridge.

Reply:
How sure are you?  I'd say keep this 1 line unless this fact is verified.

Reply:
I am sure since there's been no change in _source_ between compiler-bridge_2.13.0-M5 (prior to this PR) vs compiler-bridge_2.13 (aka _2.13.0-RC1).